src/styles/style.css

```diff
@@ -0,0 +1,250 @@
+}
+
+.is-active,
+.nav__link[data-qa='nav-hover']:hover {
+  color: #00acdc;
+  transition: color 300ms ease;
+}
+
+.is-active::after {
+  content: '';
+  width: 100%;
+  height: 4px;
+  border-radius: 2px;
+  display: block;
+  bottom: 0;
+  position: absolute;
+  background-color: #00acdc;
+}
+
+.card__rate-stars {
+  display: flex;
+}
+
+.card__rate-stars-star {
+  width: 16px;
+  height: 16px;
+  background: url(../images/star.svg) no-repeat center;
+  margin-right: 4px;
+}
+
+.card__rate-stars--1 .card__rate-stars-star:nth-child(-n + 1) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card__rate-stars--2 .card__rate-stars-star:nth-child(-n + 2) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card__rate-stars--3 .card__rate-stars-star:nth-child(-n + 3) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card__rate-stars--4 .card__rate-stars-star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card__rate-stars--5 .card__rate-stars-star:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  padding-left: 16px;
+}
+
+.card__photo {
+  width: 160px;
+  height: 134px;
+  background-size: contain;
+  background-repeat: no-repeat;
+  padding-top: 32px;
+  padding-left: 3px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  display: block;
+  width: 166px;
+  padding-top: 40px;
+  margin: 0;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  padding-top: 4px;
+  margin: 0;
+  display: block;
+  width: 98px;
+}
+
+.card__rate {
+  display: flex;
+  width: 166px;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 16px;
+}
+
+.card__rate-text {
+  display: flex;
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+  width: 166px;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  text-align: right;
+  color: #060b35;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  height: 40px;
+  margin-top: 16px;
+  box-sizing: border-box;
+  background-color: #00acdc;
+  border-radius: 5px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+.card__button-text {
+  display: block;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+}
+
+.card__button:hover {
+  background-color: #34568b;
+  border: 1px solid #00acdc;
+  color: #00acdc;
+  transition:
+    color 300ms linear,
+    background-color 300ms linear,
+    border 300ms linear;
```

Comment:
The hover effect on `.card__button` and `.card__button[data-qa='card-hover']:hover` are very similar. Consider combining these rules to reduce redundancy and improve maintainability.